client/state/data-layer/wpcom/comments/index.js

```diff
@@ -119,7 +118,7 @@ export const announceFailure = ( { siteId, postId } ) => ( dispatch, getState )
 	const postTitle = post && post.title && post.title.trim().slice( 0, 20 ).trim().concat( '…' );
 	const error = postTitle
 		? translate( 'Could not retrieve comments for “%(postTitle)s”', { args: { postTitle } } )
-		: translate( 'Could not retrieve comments for requested post' );
+		: translate( 'Could not retrieve comments for post' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.